docs/features/authentication.rst

```diff
@@ -0,0 +1,14 @@
+.. _berks:authentication
+
+Authentication
+==============
+
+Fresh start
+-----------
+
+If you have not started the server without authentication and a persistent data store configured, just start it with authentication enabled and a conf-root directory. On the first start the admin, chef-webui, chef-validator keys will be saved to the directory given as conf-root. The admin client is not created yet, but you can use the admin.pem as 'admin' client with knife to create it (knife client create admin). This will give you a new private key for the user, you'll have to replace the previous with it.
```

Comment:
I'm not exactly sure what you mean here by "the admin client is not created yet". The admin user should be created on startup, and it shouldn't let you create another client named admin.